### ISSUE TITLE
Show beginning of a file

### ISSUE THREAD

**Issue by seijikohara** [marktext/marktext#2694]
### Describe your feature request

When a file is displayed, I want it to be displayed from the top.

### What problem does this feature solve? [optional]

Currently, last line is displayed when a file is opened.
I would like to be able to set Preferences to show either beginning or end of a file.

**Comment by davidvidmar** [comment]
Same behavior can be oserved, if file is changed outside of the editor. Reload happens, but cursor is set on the last line. It should be where it was when reload happens, e.g. on top if file was recently opened in marktext.

**Comment by simonnier** [comment]
I can not agree more. Showing the last line is really annoying. It is almost a year passed, at least there should be an option for users. Hope this feature can be added.

**Comment by Definitely-Legit** [comment]
I completely agree. For me it doesn't make sense to open files at the end by default. I think there should be an option to change this in preferences.

**Comment by Jocs** [comment]
Thanks for the report, and to everyone who chimed in. This was a behavior of MarkText's old editor engine (muyajs), which placed the caret at the end of the document on open. MarkText has since been rewritten on a new editor engine (@muyajs/core), and opening a file now starts at the top with the caret at the beginning of the first block (no scroll-to-end logic remains in the open path). Because the original problem no longer reproduces, I'm closing this as resolved by the engine migration. If you still see a file open scrolled to the end on a current build, please reopen with the file and your MarkText version and we'll take another look.
